static/css/restyle/restyle.less

```diff
@@ -2050,3 +2050,51 @@ body.editor-tools {
   }
 
 }
+
+/* Source/Diff viewer */
+body.file-viewer {
+  pre, code, kbd, tt, samp, tt {
+    font-family: "andale mono", monospace !important;
```

Comment:
The fact that we have to overwrite the font with `!important` for most anything really bothers me. I guess this is due to 25443ef13496340cd0085d8a605a3b80304a2eee where the `*` rule has specificity 0, overruling even a tag selector. @muffinresearch do you think your rule in that commit can be written differently?

That aside, in this file, we should probably be using `@mono-stack` instead of a specific font name.